### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by twospheres**

### CODE REVIEW

**Review by henrikbulie — APPROVED** [review]
LGTM!

File: .github/workflows/release-version.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  release-version:
+    name: Create SemVer releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Create version releases
+        uses: climpr/semver-release@v0
```
Comment (audhage) [inline .github/workflows/release-version.yaml:38]: Should be @v1

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+  release-version:
+    name: Create SemVer releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Create version releases
+        uses: climpr/semver-release@v0
```
Comment (audhage) [inline README.md:48]: Same here

**Review by audhage — CHANGES_REQUESTED** [review]

File: .github/workflows/release-version.yaml (hunk 1 above)
Comment (twospheres) [reply .github/workflows/release-version.yaml:38]: proceeding with v0 as agreed. v1 will come after testing succeeds

File: README.md (hunk 2 above)
Comment (twospheres) [reply README.md:48]: proceeding with v0 as agreed. v1 will come after testing succeeds

**Review by audhage — APPROVED** [review]

**Pull request merged**